Traducir.Web.Net/Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Traducir.Api.Services;
+using Traducir.Core.Services;
+using Traducir.Web.Net.Models;
+using Traducir.Web.Net.ViewModels.Home;
+
+namespace Traducir.Web.Net.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly IStringsService stringsService;
+
+        public HomeController(IStringsService stringsService)
+        {
+            this.stringsService = stringsService;
+        }
+
+        public async Task<IActionResult> Index()
```

Comment:
if you're not doing anything with the result of an `await`, then you don't need to `await` for it :)

so this should be

```c#
public Task<IActionResult> Index()
{
  return Filters()
}
```

that prevents the compiler from adding the `async` state machine

Reply:
Damn. My .NET is rusting. 😛

Reply:
Well that didn't work, it throws an error:

> This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

Reply:
remove `async` from the method signature as well... it could be

```c#
public Task<IActionResult> Index() => Filters();
```

Reply:
I'm assuming you did something like this:

```c#
public async Task<IActionResult> Index()
{
  return Filters()
}
```

instead of

```c#
public Task<IActionResult> Index()
{
  return Filters()
}
```

Reply:
Yep, that was it! 👌